### ISSUE TITLE
SPRINT 5 :: ARIA Tags

### ISSUE THREAD

**Issue by Kristiannmiller** [ericberglund117/Refactor-tractor-WC-EB-KM-CD#58]
ARIA tags should be used (ONLY if needed / appropriate)
